CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## v1.1.1
+
+- [core] added static function to fix Windows filepaths that are using `\` instead of `/`
+- [plugin-ext] fixed custom Icon Themes not working
```

Comment:
there is one item for each PR with a reference and sub points if needed, please see records below for an example